core/pom.xml

```diff
@@ -616,6 +626,52 @@
         <script.extension>.sh</script.extension>
       </properties>
     </profile>
+    <profile>
+      <id>opentelemetry-reporter</id>
```

Comment:
It seems too long. Shall we use `opentelemetry` instead of `opentelemetry-reporter`?